alpakaConfig.cmake

```diff
@@ -1005,6 +1007,8 @@ IF(NOT TARGET "alpaka")
         INTERFACE cxx_std_${ALPAKA_CXX_STANDARD}
     )
 
+    target_compile_definitions("alpaka" INTERFACE ALPAKA_CXX_STANDARD=${ALPAKA_CXX_STANDARD})
```

Comment:
We should not add this here generally. `target_compile_features` (that we are already using) should be the way forward.
If this is necessary, the inheriting target probably did something wrong.